converter/biostudies.py

```diff
@@ -21,6 +23,44 @@ def _parse_name(*args):
     return name_element[0] if name_element and position == 1 else name_element
 
 
+PUBLICATION_SPEC = {
+    "type": "Publication",
+    "on": 'publications',
+    "attributes_to_include": {
+        'authors': "Authors",
+        'title': "Title",
+        'doi': 'doi',
+        'url': 'URL'
+    },
+    "attribute_handler": {
+        'authors': array_to_string
```

Comment:
nitpick: we could have consistent string quotes for this and the other specs

Reply:
that is true. It is a bad typo.